autosklearn/automl.py

```diff
@@ -1457,6 +1671,8 @@ def fit(
         dataset_name: Optional[str] = None,
         only_return_configuration_space: bool = False,
         load_models: bool = True,
+        task: Optional[int] = None,
```

Comment:
Yes, but the signature of the parent and child be the same? I think this will be a mypy error, but in my particular case I had to pass `task` argument and got a problem because the base class was missing this argument.

Reply:
That's weird because it's immediately overwritten

Reply:
Oh, yes, this is no longer needed if I directly specify the arguments. This was needed when I was using **kwargs. I have updated this.